xonsh/platform.py

```diff
@@ -232,6 +232,9 @@ def pathsplit(p):
     without a drive.
     """
     n = len(p)
+    if n == 0:
+        # lazy object seps does not get initialized when n is zero
+        return "", ""
```

Comment:
Shouldn't this be a single string in a tuple? `("",) `

Reply:
If I understand correctly, `pathsplit()` is supposed to have the same behavior as `os.path.split()`, which returns `("", "")` for an empty string.